server/datastore/mysql/software_installers.go

```diff
@@ -313,13 +313,14 @@ func (ds *Datastore) addSoftwareTitleToMatchingSoftware(ctx context.Context, tit
 	return ctxerr.Wrap(ctx, err, "adding fk reference in software to software_titles")
 }
 
+const (
```

Comment:
Was thinking a full-on type here rather than just a string, similar to what I did for host lock. This code path isn't inline with ingestion so we can afford to take the cycles to be more strict here.

Reply:
Haha I went back and forth on full type or not, sgtm!